src/main/java/org/isf/permissions/manager/GroupPermissionManager.java

```diff
@@ -21,19 +21,17 @@
  */
 package org.isf.permissions.manager;
 
-import java.util.List;
-
-import org.isf.menu.model.UserGroup;
 import org.isf.permissions.model.GroupPermission;
-import org.isf.permissions.model.Permission;
 import org.isf.permissions.service.GroupPermissionIoOperations;
 import org.isf.utils.exception.OHServiceException;
 import org.springframework.stereotype.Component;
 
+import java.util.List;
```

Comment:
Please follow the project standards for ordering of imports; see https://github.com/informatici/openhospital-core/blob/develop/.ide-settings/eclipse/OpenHospital.importorder